src/python/ImpactX.cpp

```diff
@@ -29,7 +30,34 @@ namespace impactx {
     struct Config {};
 }
 
-void init_ImpactX(py::module& m)
+namespace detail
+{
+    /** Helper Function for Property Getters
```

Comment:
We can check in the future if we find a reasonable location to set default parameters without setting them multiple times throughout the code.

Maybe we need a `ReadParameters`-like function called `set_default_parameters` after all... :-S